config/jobs/kubernetes/sig-release/release-branch-jobs/1.27.yaml

```diff
@@ -1755,36 +1755,6 @@ presubmits:
             memory: 12Gi
         securityContext:
           privileged: true
-  - always_run: false
-    branches:
-    - release-1.27
-    cluster: k8s-infra-prow-build
-    context: pull-kubernetes-verify-strict-lint
```

Comment:
is it ok to remove it from the release 1.27 branch?

Reply:
It was optional and experimental, so yes, I think it can be removed.